doc/spec/Types.md

```diff
@@ -155,12 +161,59 @@ var x = undefined;      // Same as x: any = undefined
 var e: Undefined;       // Error, can't reference Undefined type
 ```
 
+#### Literal types { #literal-types }
+
+TypeScript supports string, number and boolean literal types, but not regular expression literal types.

```

Comment:
Get rid of ", but not regular expression literal types." it's unnecessary

Reply:
Done, and I noticed that I missed yet another oxford comma.